addons/mail/models/mail_message_translation.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class MessageTranslation(models.Model):
+    _name = "mail.message.translation"
+    _description = "Message Translation"
+
+    message_id = fields.Many2one(
+        "mail.message",
+        "Message",
+        required=True,
+        index=True,
+        ondelete="cascade",
+    )
+    target_lang = fields.Char(
+        "Target Language",
+        help="Shortened language code used as target for the translation request.",
+        index=True,
+    )
+    source_lang = fields.Char("Source Language", help="Result of the language detection based on its content.")
+    value = fields.Char(
```

Comment:
I disagree, the model name is explicit enough. The problem is that if I create a variable referencing a mail.message.translation recordset anywhere else it will end up looking like "translation.translated_text" or "translation.translation" which is redundant.

Reply:
then how about body? to mirror the corresponding value on mail.message